documentation/package.json

```diff
@@ -30,7 +30,7 @@
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "@hashicorp/consul-ui-toolkit": "^0.1.0",
-    "@hashicorp/design-system-components": "^2.8.1",
+    "@hashicorp/design-system-components": "2.8.1",
```

Comment:
pinned this at 2.8.1 for now as 2.9 is breaking embroider. Fix for that will be released by DS soon™.